src/encoded/genomic_data_service.py

```diff
@@ -0,0 +1,136 @@
+import requests
+
+REGISTRY_DATA_SERVICE = 'genomic_data_service'
+RNA_GET_FILTERS = ['assayType', 'annotation']
+RNA_GET_ENDPOINT = '/expressions/bytes'
+
+# react component orders columns by "the position" in the hash map
+RNA_GET_COLUMNS = {
+    'featureID': {'title': 'Feature ID'},
+    'tpm': {'title': 'Counts (TPM)'},
+    'fpkm': {'title': 'Counts (FPKM)'},
+    'assayType': {'title': 'Assay (RNA SubType)'},
+    'libraryPrepProtocol': {'title': 'Experiment'},
+    'expressionID': {'title': 'File'},
+    'annotation': {'title': 'Genome Annotation'}
+}
+
```

Comment:
Some formatting nits, not a big deal. Usually try to do two lines between function, one line between methods, and remove extra lines.

https://www.python.org/dev/peps/pep-0008/#blank-lines